### ISSUE TITLE
Logging issues on HMR accept

### ISSUE THREAD

**Issue by verydanny** [shellscape/webpack-plugin-serve#101]
- Webpack Version: 4.29
- Operating System (or Browser): Chrome 71
- Node Version: 11.6.0
- webpack-plugin-serve Version: 0.5.0

### How Do We Reproduce?

webpack.config.js
```js
const { WebpackPluginServe } = require('webpack-plugin-serve')
const HtmlWebpackPlugin = require('html-webpack-plugin')
const path = require('path')

const outputDir = path.resolve(process.cwd(), 'dist')

module.exports = {
  mode: 'development',
  entry: [path.resolve(__dirname, 'src'), 'webpack-plugin-serve/client'],
  output: {
    path: outputDir,
    filename: 'bundle.js',
    publicPath: '/',
  },
  module: {
    rules: [
      {
        test: /\.tsx?$/,
        exclude: /node_modules/,
        loader: 'babel-loader',
        options: {
          cacheDirectory: true,
        },
      },
    ],
  },
  plugins: [
    new WebpackPluginServe({
      static: outputDir,
      progress: 'minimal',
    }),
    new HtmlWebpackPlugin({
      template: path.resolve(__dirname, 'template.html'),
    }),
  ],
  watch: true,
  resolve: {
    extensions: ['.js', '.jsx', '.json', '.tsx', '.ts'],
  },
}
```

src/index.ts
```ts
console.log('Hello')

if (module.hot) {
  module.hot.accept()
}
```

### Expected Behavior

No error? My guess is it's something to do with with `lib/client/hmr.js` on line 11:

```js
const { error, info, refresh, warn } = require('./log');
```
should be?

```js
const { error, info, refresh, warn } = require('./log').log;
```
My guess is that { silent } in client is failing this way as well. 

### Actual Behavior

Modify console.log('Hello') to anything else and you get the error:
`Uncaught (in promise) TypeError: info is not a function at replace (webpack:///(:55555/webpack)-plugin-serve/lib/client/hmr.js?:64:7)`

**Comment by shellscape** [comment]
Dang. Thanks for the heads up. I'll get a fix out shortly.

**Comment by shellscape** [comment]
Resolved in 1dbf4aa6e2acc789c8e5c312e270c7e6345b0e52 Cheers
